content/documentation/getting-started/index.adoc

```diff
@@ -56,26 +59,30 @@ oc login -u system:admin
 ----
 
 
-=== Install Istio
+== Install Kiali via Istio or Maistra
+
+icon:lightbulb[size=1x]{nbsp} Both Istio and Maistra include Kiali distributions.
 
-Make sure your cluster environment has Istio installed. For details on how to install Istio, see the link:https://istio.io/docs/setup/[Istio Setup docs].  If you are running on OpenShift and prefer Maistra, see link:https://maistra.io/docs/getting_started/install/[Maistra Setup docs].
+If you prefer to use the latest Kiali version then choose not to install Kiali during the Istio or Maistra installation procedure. Complete the Istio or Maistra installation and then link:#_install_kiali_latest[Install Kiali Latest].
 
-icon:lightbulb[size=1x]{nbsp}Both Istio and Maistra include Kiali distributions.
+To install Kiali as part of the Istio installation just follow the link:https://istio.io/docs/setup/[Istio Setup docs].  If you are running on OpenShift and prefer Maistra, see link:https://maistra.io/docs/getting_started/install/[Maistra Setup docs].  You may then continue to link:#_open_the_ui[Open the UI].
 
-If the Kiali version packaged with Istio or Maistra meets your needs then install Kiali using the Istio or Maistra installation procedure. You may then continue to link:#_open_the_ui[Open the UI].
 
-If you prefer to use the latest Kiali version then do not install Kiali during the Istio or Maistra installation procedure. Complete the Istio or Maistra installation and then install Kiali using the steps below. Note that the Kiali installation script documented below will ask to delete an existing Kiali installation, if found.
+=== Upgrade the Istio or Maistra Version of Kiali
 
+The version of Kiali packaged with Istio or Maistra may not contain the most recent features and fixes. To upgrade to the latest version of Kiali first link:#_wipe_kiali[Wipe Kiali] and then proceed to link:#_install_kiali_latest[Install Kiali Latest].
```

Comment:
Word choice - I would use "remove" rather than "wipe".

Reply:
This is a little more involved.  There is Kiali the app, and then there is everything to do with Kiali: the app, the operator, roles, etc.  I called this Wipe to indicate that it deleted everything.  I've change the two Uninstall anchors to "Uninstall Kiali Only" (i.e. the app).  And "Uninstall Kiali Operator and Kiali" (i.e. wipe).

Reply:
That sounds like a good solution.